spec/bowling_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'bowling'
+
```

Comment:
Couldn't see a good place to put this. I see in your code you have a guard clause, may want to right a test for it to see if it works as you intend it to?